examples/cms-sanity/README.md

```diff
@@ -0,0 +1,140 @@
+# A statically generated blog example using Next.js and Sanity
+
+This example showcases Next.js's [Static Generation](/docs/basic-features/pages.md) feature using [Sanity](https://www.sanity.io/) as the data source.
+
+## Demo
+
+### [https://next-blog-sanity.now.sh/](https://next-blog-sanity.now.sh/)
+
+## How to use
+
+### Using `create-next-app`
+
+Execute [`create-next-app`](https://github.com/zeit/next.js/tree/canary/packages/create-next-app) with [npm](https://docs.npmjs.com/cli/init) or [Yarn](https://yarnpkg.com/lang/en/docs/cli/create/) to bootstrap the example:
+
+```bash
+npm init next-app --example cms-sanity cms-sanity-app
+# or
+yarn create next-app --example cms-sanity cms-sanity-app
+```
+
+### Download manually
+
+Download the example:
+
+```bash
+curl https://codeload.github.com/zeit/next.js/tar.gz/canary | tar -xz --strip=2 next.js-canary/examples/cms-sanity
+cd cms-sanity
+```
+
+## Configuration
+
+### Step 1. Create an account and a project on Sanity
+
+First, [create an account on Sanity](https://sanity.io).
+
+After creating an account, install the Sanity cli from npm `npm i -g @sanity/cli`.
+
+### Step 2. Create a new Sanity project
+
+In a separate folder run `sanity init` to initialize a new studio project.
+
+This will be where we manage our data.
+
+When going through the init phase make sure to select **Yes** to the **Use the default dataset configuration** step and select **Clean project with no predefined schemas** for the **Select project template** step.
+
+After initializing, get the `projectId` value from the `sanity.json` file that was generated for the next step.
+
+### Step 3. Set up environment variables
+
+First, copy the `.env.example` file in this directory to `.env` (which will be ignored by Git):
+
+```bash
+cp .env.example .env
+```
+
+Then set each variable on `.env`:
+
+- `NEXT_EXAMPLE_CMS_SANITY_PREVIEW_SECRET` can be any random string (but avoid spaces), like `MY_SECRET` - this is used for [the Preview Mode](/docs/advanced-features/preview-mode.md).
+- `SANITY_PROJECT_ID` the `projectId` from the previous step
+
+Your `.env` file should look like this:
+
+```bash
+NEXT_EXAMPLE_CMS_SANITY_PREVIEW_SECRET=...
+SANITY_PROJECT_ID=...
+```
+
+### Step 4. Copy the schema file
+
+After initializing your Sanity studio project there should be a `schemas` folder.
+
+To get up and running we want to copy the schema file from [`./schemas/schema.js`](./schemas/schema.js) to the studio project to use for creating/editing content.
+
+### Step 5. Populate Content
+
+To add some content go to your Sanity studio project where we added the schema file and run `sanity start`.
```

Comment:
If we say "where we added the schema file" then it might confuse them (they don't want to run `sanity start` in the `schema` directory.

```suggestion
To add some content go to your Sanity studio project directory and run `sanity start`.
```